app/managers.py

```diff
@@ -0,0 +1,34 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.table_name = "actors"
+        self._connection = sqlite3.connect("cinema")
+
+    def all(self) -> list:
+        data = self._connection.execute("SELECT id, first_name, last_name "

```

Comment:
Better to use select * because when you will change Actor model. You must change this code also